components/table/table-col-sort-button.js

```diff
@@ -37,30 +37,40 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				align-items: center;
 				background-color: transparent;
 				border: none;
+				box-sizing: border-box;
 				color: inherit;
 				cursor: pointer;
 				display: inline-flex;
 				font-family: inherit;
 				font-size: inherit;
+				gap: 12px;
+				height: calc(var(--d2l-table-cell-height) + var(--d2l-sortable-button-height));
```

Comment:
I find this a bit confusing because `--d2l-sortable-button-height` also itself depends on `--d2l-table-cell-height`. Is this something to do with the dropdown button that we should be leaving out of this PR or is unrelated? If not, can this be done a bit more clearly?

I also noticed that `--d2l-table-cell-height` isn't actually used in connection with the header (it's overwritten with `27px`) so it might be best to not involve it.